### ISSUE TITLE
FR: Negative filtering on admin console's machine view

### ISSUE THREAD

**Issue by darshinimashar** [tailscale/tailscale#3093]
### Tell us about your idea!

Negative filtering on admin console's machine view

### What are you trying to do?

From User:

On the login.tailscale.com/admin page, it would be nice to be able to do one of the following, or all:
* use "!github" to hide hosts that have "github" in their name
* use "-github" for same effect?
* use "-type:ephemeral" to hide ephemeral clients

Basically the gist is an easy way to hide a class of ephemeral clients that I mostly don't care about from a quick glance standpoint.

<img src="https://frontapp.com/assets/img/favicons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_3o87l)

**Comment by bradfitz** [comment]
By "Github dashboard" you mean the admin console's machine view?

/cc @tailscale/frontend

**Comment by colemickens** [comment]
Yes. original ask was just "filtering on dashboard" but I did mean the admin console machine list view. Super minor "issue", just a nice-to-have. Thanks.

**Comment by sonovawolf** [comment]
Valuable feedback for https://github.com/tailscale/tailscale/issues/2540
